README.md

```diff
@@ -1 +1,46 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
```

Comment:
- [ ] [OPTIONAL] You should have 2 files committed: a .html and .css file, kindly consider removing other files in the PR